lib/sanitiser/strategy.rb

```diff
@@ -0,0 +1,18 @@
+module Sanitiser
+  class Strategy
+    class SanitisingError < StandardError; end
+
+    class << self
+      def call(input, sanitize_null_bytes: false)
+        input
+          .force_encoding(Encoding::ASCII_8BIT)
+          .encode!(Encoding::UTF_8)
```

Comment:
Isn't this also just the default?

Reply:
Please, see my comment above and the link to the strategy there.